### PR TITLE
Fix emulation for GRiSP 2

### PR DESCRIPTION
**Pull request by ziopio**

### CODE REVIEW

File: src/grisp_rtems.erl [hunk 1]
```diff
@@ -102,7 +102,7 @@ pwrite(DevicePath, Buffer, Offset) ->
 %--- Callbacks -----------------------------------------------------------------
 
 % @private
-on_load() -> ok = erlang:load_nif(atom_to_list(?MODULE), 0).
+on_load() -> ?NIF_LOAD.
```
Comment (ziopio) [inline src/grisp_rtems.erl:105]: @eproxus this was the problem

Comment (eproxus) [reply src/grisp_rtems.erl:105]: Ah! Nice catch!

**Comment by ziopio** [comment]
CI is now running the `pmod_nav_tests` using `grisp_emulation` `0.2.0`

File: erlang_ls.config
Comment (eproxus) [inline erlang_ls.config:1]: This file can probably be deleted, or reverted? Not sure why I committed this 😅

**Review by eproxus — APPROVED** [review]
Looks good. I think the Erlang LS config can be reverted though (unless it breaks something?)

File: erlang_ls.config
Comment (ziopio) [reply erlang_ls.config:1]: is there on master so I'll probably revert it

Comment (ziopio) [reply erlang_ls.config:1]: Checked all files, no complains from Erlang LS

**Pull request merged**